src/Discord.Net.Core/Entities/Activities/GameAssets.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Discord
+{
+    public class GameAssets
```

Comment:
Given the only context that this pairing occurs is in a Rich Presence, I would suggest removing this class and having properties for each asset separately.